src/test/java/seedu/mark/testutil/ReminderBuilder.java

```diff
@@ -0,0 +1,71 @@
+package seedu.mark.testutil;
+
+import static seedu.mark.testutil.TypicalBookmarks.ALICE;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import seedu.mark.model.bookmark.Bookmark;
+import seedu.mark.model.reminder.Note;
+import seedu.mark.model.reminder.Reminder;
+
+
+/**
+ * A utility class to help with building Reminder objects.
+ */
+public class ReminderBuilder {
+    public static final String DEFAULT_NOTE = "See Website";
+    public static final Bookmark DEFAULT_BOOKMARK = ALICE;
+    public static final String DEFAULT_TIME = "12/12/2020 1800";
+    private static final String DATE_FORMATTER = "dd/MM/yyyy HHmm";
```

Comment:
You can just use the formatter in the Reminder class. These two should be the same

Reply:
I haven't push the parser class and want to leave the formatter to parser.

Reply:
Ok